src/pnettest.rs

```diff
@@ -319,7 +319,7 @@ fn get_test_interface() -> datalink::NetworkInterface {
 }
 
 // FIXME Find a way to test this with netmap
-#[cfg(all(not(feature = "appveyor"), not(feature = "netmap")))]
+#[cfg(all(not(feature = "appveyor"), not(feature = "netmap"), not(feature = "pcap")))]
```

Comment:
These used to work with pcap, do they not any more?

Reply:
No, now that `pcap` is a backend.